src/fsharp/absil/il.fs

```diff
@@ -4254,13 +4256,27 @@ and refs_of_token s x =
     | ILToken.ILMethod mr -> refs_of_mspec s mr
     | ILToken.ILField fr -> refs_of_fspec s fr
 
+and refs_of_attrib_elem s (e: ILAttribElem) =
+    match e with
+    | Type (Some ty) -> refs_of_typ s ty
+    | TypeRef (Some tref) -> refs_of_tref s tref
+    | Array (ty, els) ->
+        refs_of_typ s ty 
+        refs_of_attrib_elems s els
+    | _ -> ()
```

Comment:
It looks like the problem remains here due to the literal reference is erased. We only have a literal value represented by the underlying type here, instead of the reference like `FontStyle.Bold` that could be used to add a type reference to the output assembly.